griffin-metric/pom.xml

```diff
@@ -17,24 +17,23 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
-xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.apache</groupId>
-        <artifactId>apache</artifactId>
-        <version>17</version>
-        <relativePath />
+        <groupId>org.apache.griffin</groupId>
+        <artifactId>griffin</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
     </parent>
 
-    <groupId>org.apache.griffin</groupId>
+    <groupId>org.apache.griffin.metric</groupId>
```

Comment:
I think we'd better to keep the group id of all components to be `org.apache.griffin`, otherwise there will be some trouble when locating artifacts in Maven Central.

Reply:
Splitting modules makes sense to me, but I don't think it's necessary to change the group id, using different artifact id should be enough here.

Reply:
@whhe , agree with you. I'll keep them using a same group id.

make another PR to revise. close this one.

appreciate your comment.